test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length+1, sizeof(char));
+    if (!test_str) {
+        printf("unable to allocate memory\n");
```

Comment:
And please do the same for other error messages. Error messages must be print into stderr in this example.